app/src/components/Admin/UsersTableAdmin.tsx

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
```

Comment:
I feel like this is a bad rule and maybe we should hear what people think about disabling it 🤔

Reply:
Just to be clear, that is not something that needs to be done for this PR, that was more of a general comment